### ISSUE TITLE
[onvista] Additional account statements for PR #1072

### ISSUE THREAD

**Issue by codeofsumit** [tresorone/tresor-import#1146]
## Description
> Related to #1072

* 20161231_Kontoauszug.pdf: Contains a market order for buy and sell, also this document has an old format
* 20170630_Kontoauszug.pdf: Can't be parsed (status code 1)
* 20171229_Kontoauszug.pdf: Can't be parsed (status code 5)
* 20200331_Kontoauszug.pdf: Contains a dividend payment (but the amount is not the gross, how should we handle this? Did we need a isNetPayment Field?)

## Documents
<details><summary>20161231_Kontoauszug.pdf</summary><p>

```json
[
  [
    "Ihre Kontonummer : 123456789 : Customer Cash Account  EUR",
    "KONTOAUSZUG Nr. 2 per 31.12.2016",
    "Alter Saldo",
    "Neuer Saldo",
    "0,00",
    "H",
    "0,00",
    "H",
    "Guthaben",
    "sind",
    "als",
    "Einlagen",
    "nach",
    "Maßgabe",
    "des",
    "Einlagensicherungsgesetzes",
    "entschädigungsfähig.",
    "Nähere",
    "Informationen",
    "können",
    "dem",
    "\"Informationsbogen",
    "für",
    "den",
    "Einleger\",",
    "der",
    "unter",
    "www.onvista-bank.de",
    "im",
    "Menü",
    "\"Service",
    "-  Formulare\"",
    "abgelegt",
    "ist,",
    "entnommen",
    "werden.",
    "Unsere Postanschrift:",
    "Postfach 10 08 60",
    "60008 Frankfurt",
    "Ihre Kundenbetreuer:",
    "Telefon",
    "(069) 7107530",
    "Telefax",
    "(069) 7107100",
    "Für Überweisungen:",
    "Kreditinstitut",
    "OnVista Bank GmbH",
    "Bankleitzahl",
    "537 456 63",
    "Für EU-Standardüberweisungen:",
    "BIC",
    "BOURDEFF",
    "IBAN",
    "LN42806308181623529769",
    "Herr",
    "Osoqh Thdshl",
    "Uuqkwfxsyajrcqßv 11",
    "05197 Cvnlpcmx Nn Qfrqgbfq",
    "Seite 1 von 2"
  ],
  [
    "Ihre Kontonummer : 123456789 : Customer Cash Account  EUR",
    "KONTOAUSZUG Nr. 2 per 31.12.2016",
    "B.Datum",
    "Valuta",
    "Bezeichnung",
    "Soll",
    "Haben",
    "08.12.",
    "08.12.",
    "REF: 123456789",
    "150,00",
    "Überweisungseingang SEPA Max Mustermann",
    "NKSKDW34WTX / BX80556323039352045071",
    "09.12.",
    "12.12.",
    "REF: 123456789",
    "64,50-",
    "Wertpapierkauf",
    "AIR BERLIN PLC",
    "EO -,25",
    "STK: 100",
    "ISIN: GB00B128C026",
    "ABR: 123456789",
    "15.12.",
    "14.12.",
    "REF: 123456789",
    "0,89-",
    "Registrierung Namensaktie",
    "AIR BERLIN PLC",
    "EO -,25",
    "STK: 100",
    "ISIN: GB00B128C026",
    "ABR: 123456789",
    "22.12.",
    "23.12.",
    "REF: 123456789",
    "51,95",
    "Wertpapierverkauf",
    "AIR BERLIN PLC",
    "EO -,25",
    "STK: 100",
    "ISIN: GB00B128C026",
    "ABR: 6839141234567894920161221",
    "29.12.",
    "29.12.",
    "REF: 123456789",
    "136,56-",
    "Überweisungausgang SEPA Max Mustermann",
    "MSAQBNTRDZO / ZZ10099680793483292691",
    "Reture OnVista",
    "CSFS_Ak: G804179670",
    "QXF_Lj: 324564435",
    "Seite 2 von 2",
    "Alter Saldo",
    "0,00",
    "Neuer Saldo",
    "0,00"
  ]
]

```

</p></details>

<details><summary>20170630_Kontoauszug.pdf</summary><p>

```json
[
  [
    "Kontoauszug Nr. 2017 / 2 und Rechnungsabschluss zum 30.06.2017",
    "EUR - Verrechnungskonto: 123456789",
    "Alter Saldo",
    "Neuer Saldo",
    "0,00",
    "H",
    "0,00",
    "H",
    "Guthaben",
    "sind als Einlagen",
    "nach",
    "Maßgabe",
    "des Einlagensicherungsgesetzes",
    "entschädigungsfähig.",
    "Nähere",
    "Informationen",
    "können",
    "dem",
    "\"Informationsbogen",
    "für den Einleger\",",
    "der unter",
    "www.onvista-bank.de",
    "im Menü",
    "\"Service",
    "- Formulare\"",
    "abgelegt",
    "ist, entnommen",
    "werden.",
    "Einwendungen",
    "wegen",
    "Unrichtigkeit",
    "oder",
    "Unvollständigkeit",
    "eines",
    "Rechnungsabschlusses",
    "sind gem.",
    "Nr. 7",
    "Abs.",
    "2 unserer",
    "Allgemeinen",
    "Geschäftsbedingungen",
    "spätestens",
    "vor Ablauf",
    "von sechs",
    "Wochen",
    "nach",
    "Zugang",
    "des Rechnungsabschlusses",
    "zu erheben.",
    "Sofern",
    "Sie Ihre Einwendungen",
    "schriftlich",
    "geltend",
    "machen,",
    "richten",
    "Sie diese",
    "bitte",
    "unter",
    "Angabe",
    "der oben",
    "genannten",
    "Kontonummer",
    "an die OnVista",
    "Bank",
    "GmbH,",
    "Innenrevision,",
    "Postfach",
    "10 08 60, 60008",
    "Frankfurt",
    "am Main.",
    "Es genügt",
    "die Absendung",
    "innerhalb",
    "der Sechs-Wochen-Frist.",
    "Das Unterlassen",
    "rechtzeitiger",
    "Einwendungen",
    "gilt als",
    "Genehmigung.",
    "Bei Zahlungen",
    "größer",
    "12.500",
    "EUR",
    "bitte Meldepflichten",
    "nach",
    "dem Außenwirtschaftsrecht",
    "beachten.",
    "Unsere Postanschrift:",
    "Postfach 10 08 60",
    "60008 Frankfurt",
    "Kundenservice:",
    "Telefon:",
    "+49 (0) 69 7107 - 530",
    "Fax",
    "+49 (0) 69 7107 - 100",
    "Mo-Fr 8:00 - 19:00 Uhr",
    "bkpjqbe@shnowyh-zbbk.le",
    "Ihre Bankverbindung für Überweisungen:",
    "IBAN",
    "NL65131828278135432629",
    "BIC",
    "BOURDEFF",
    "Herr",
    "Iggei Fvtshj",
    "Ieedefmvhuedsbßw 72",
    "79831 Ajagjcvb Gu Qadpuvms",
    "Seite 1 von 2"
  ],
  [
    "Kontoauszug Nr. 2017 / 2 und Rechnungsabschluss zum 30.06.2017",
    "EUR - Verrechnungskonto: 123456789",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "05.04.",
    "05.04.",
    "REF: 123456789",
    "0,70-",
    "Portogebühren",
    "Portogebuehren 03/17",
    "19.04.",
    "19.04.",
    "REF: 123456789",
    "0,70+",
    "Überweisungseingang SEPA Max Mustermann",
    "KLHERK82BSU / XX51263799641771261017",
    "Seite 2 von 2",
    "Alter Kontostand vom 31.03.17",
    "0,00+",
    "Neuer Kontostand zum 30.06.2017",
    "0,00+"
  ]
]

```

</p></details>

<details><summary>20171229_Kontoauszug.pdf</summary><p>

```json
[
  [
    "Kontoauszug Nr. 2017 / 4 zum 29.12.2017",
    "EUR - Verrechnungskonto: 123456789",
    "Alter Saldo",
    "Neuer Saldo",
    "0,00",
    "H",
    "560,00",
    "H",
    "Guthaben",
    "sind als Einlagen",
    "nach",
    "Maßgabe",
    "des Einlagensicherungsgesetzes",
    "entschädigungsfähig.",
    "Nähere",
    "Informationen",
    "können",
    "dem",
    "\"Informationsbogen",
    "für den Einleger\",",
    "der unter",
    "www.onvista-bank.de",
    "im Menü",
    "\"Service",
    "- Formulare\"",
    "abgelegt",
    "ist, entnommen",
    "werden.",
    "Der",
    "Ihnen",
    "angezeigte",
    "Saldo",
    "kann",
    "Kontobewegungen",
    "berücksichtigen,",
    "deren",
    "tatsächliche",
    "Wertstellung",
    "(Valuta)",
    "noch",
    "nicht",
    "erfolgt",
    "ist.",
    "Bitte",
    "beachten",
    "Sie",
    "dies",
    "bei",
    "Ihrer",
    "Gelddisposition,",
    "da",
    "ansonsten",
    "Sollzinsen",
    "/ Überziehungszinsen",
    "anfallen",
    "könnten.",
    "Das",
    "Datum",
    "der",
    "Wertstellung",
    "(Valuta)",
    "können",
    "Sie",
    "auf",
    "den",
    "folgenden",
    "Seiten",
    "zur",
    "jeweiligen",
    "Buchung",
    "entnehmen.",
    "Einwendungen",
    "wegen",
    "Unrichtigkeit",
    "oder",
    "Unvollständigkeit",
    "eines",
    "Rechnungsabschlusses",
    "sind gem.",
    "Nr. 7",
    "Abs.",
    "2 unserer",
    "Allgemeinen",
    "Geschäftsbedingungen",
    "spätestens",
    "vor Ablauf",
    "von sechs",
    "Wochen",
    "nach",
    "Zugang",
    "des Rechnungsabschlusses",
    "zu erheben.",
    "Sofern",
    "Sie Ihre Einwendungen",
    "schriftlich",
    "geltend",
    "machen,",
    "richten",
    "Sie",
    "diese",
    "bitte",
    "unter",
    "Angabe",
    "der",
    "oben",
    "genannten",
    "Kontonummer",
    "an",
    "die",
    "onvista",
    "bank,",
    "Interne",
    "Revision,",
    "Wildunger",
    "Straße",
    "6a,",
    "60487",
    "Frankfurt",
    "am",
    "Main.",
    "Es genügt",
    "die Absendung",
    "innerhalb",
    "der Sechs-Wochen-Frist.",
    "Das",
    "Unterlassen",
    "rechtzeitiger",
    "Einwendungen",
    "gilt",
    "als",
    "Genehmigung.",
    "Bei Zahlungen",
    "größer",
    "12.500",
    "EUR",
    "bitte Meldepflichten",
    "nach",
    "dem Außenwirtschaftsrecht",
    "beachten.",
    "Unsere Postanschrift:",
    "onvista bank",
    "Postfach 10 08 60",
    "60008 Frankfurt",
    "Kundenservice:",
    "Telefon:",
    "+49 (0) 69 7107 - 530",
    "Fax",
    "+49 (0) 69 7107 - 100",
    "Mo-Fr 8:00 - 19:00 Uhr",
    "cxksmkc@uonppvd-vnvq.hy",
    "Ihre Bankverbindung für Überweisungen:",
    "IBAN",
    "WV43842012707025846970",
    "BIC",
    "BOURDEFF",
    "Herr",
    "Xdxoo Wbjrkx",
    "Zo Fhlywjsyvuyta 89",
    "41802 Aoxiwege Jk Atcpsfcs",
    "Seite 1 von 2"
  ],
  [
    "Kontoauszug Nr. 2017 / 4 zum 29.12.2017",
    "EUR - Verrechnungskonto: 123456789",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "21.11.",
    "21.11.",
    "REF: 123456789",
    "200,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista",
    "NOTPROVIDED",
    "LNC508089883997808-250",
    "28.11.",
    "28.11.",
    "REF: 123456789",
    "310,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista",
    "NOTPROVIDED",
    "LZV554688126208459-050",
    "20.12.",
    "20.12.",
    "REF: 123456789",
    "350,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista",
    "NOTPROVIDED",
    "LOU461035752657998-863",
    "21.12.",
    "21.12.",
    "REF: 123456789",
    "300,00-",
    "Überweisungausgang SEPA",
    "Umbuchung",
    "QCJB_Yg: A065847866",
    "ZOC_Xs: 677307357",
    "Seite 2 von 2",
    "Alter Kontostand vom 29.09.17",
    "0,00+",
    "Neuer Kontostand zum 29.12.2017",
    "560,00+"
  ]
]

```

</p></details>

<details><summary>20200331_Kontoauszug.pdf</summary><p>

```json
[
  [
    "Kontoauszug Nr. 2020 / 1 und Rechnungsabschluss zum 31.03.2020",
    "EUR - Verrechnungskonto: 123456789",
    "Alter Saldo",
    "Neuer Saldo",
    "350,43",
    "H",
    "350,39",
    "H",
    "Guthaben",
    "sind als Einlagen",
    "nach",
    "Maßgabe",
    "des Einlagensicherungsgesetzes",
    "entschädigungsfähig.",
    "Nähere",
    "Informationen",
    "können",
    "dem",
    "\"Informationsbogen",
    "für den Einleger\",",
    "der unter",
    "www.onvista-bank.de",
    "im Menü",
    "\"Service",
    "- Formulare\"",
    "abgelegt",
    "ist, entnommen",
    "werden.",
    "Der",
    "Ihnen",
    "angezeigte",
    "Saldo",
    "kann",
    "Kontobewegungen",
    "berücksichtigen,",
    "deren",
    "tatsächliche",
    "Wertstellung",
    "(Valuta)",
    "noch",
    "nicht",
    "erfolgt",
    "ist.",
    "Bitte",
    "beachten",
    "Sie",
    "dies",
    "bei",
    "Ihrer",
    "Gelddisposition,",
    "da",
    "ansonsten",
    "Sollzinsen",
    "/ Überziehungszinsen",
    "anfallen",
    "könnten.",
    "Das",
    "Datum",
    "der",
    "Wertstellung",
    "(Valuta)",
    "können",
    "Sie",
    "auf",
    "den",
    "folgenden",
    "Seiten",
    "zur",
    "jeweiligen",
    "Buchung",
    "entnehmen.",
    "Einwendungen",
    "wegen",
    "Unrichtigkeit",
    "oder",
    "Unvollständigkeit",
    "eines",
    "Rechnungsabschlusses",
    "sind gem.",
    "Nr. 7",
    "Abs.",
    "2 unserer",
    "Allgemeinen",
    "Geschäftsbedingungen",
    "spätestens",
    "vor Ablauf",
    "von sechs",
    "Wochen",
    "nach",
    "Zugang",
    "des Rechnungsabschlusses",
    "zu erheben.",
    "Sofern",
    "Sie Ihre Einwendungen",
    "schriftlich",
    "geltend",
    "machen,",
    "richten",
    "Sie",
    "diese",
    "bitte",
    "unter",
    "Angabe",
    "der",
    "oben",
    "genannten",
    "Kontonummer",
    "an",
    "die",
    "onvista",
    "bank,",
    "Interne",
    "Revision,",
    "Wildunger",
    "Straße",
    "6a,",
    "60487",
    "Frankfurt",
    "am",
    "Main.",
    "Es genügt",
    "die Absendung",
    "innerhalb",
    "der Sechs-Wochen-Frist.",
    "Das",
    "Unterlassen",
    "rechtzeitiger",
    "Einwendungen",
    "gilt",
    "als",
    "Genehmigung.",
    "Bei Zahlungen",
    "größer",
    "12.500",
    "EUR",
    "bitte Meldepflichten",
    "nach",
    "dem Außenwirtschaftsrecht",
    "beachten.",
    "Unsere Postanschrift:",
    "onvista bank",
    "Postfach 10 08 60",
    "60008 Frankfurt",
    "Kundenservice:",
    "Telefon:",
    "+49 (0) 69 7107 - 530",
    "Fax",
    "+49 (0) 69 7107 - 100",
    "Mo-Fr 8:00 - 19:00 Uhr",
    "rwsiehj@bbupnhi-obei.xo",
    "Ihre Bankverbindung für Überweisungen:",
    "IBAN",
    "JU19212204444912831652",
    "BIC",
    "BOURDEFF",
    "Herr",
    "Fskpk Hoqetj",
    "Dy Ubeolqiahigzl 71",
    "03325 Goporkjo Dl Dicyhykc",
    "Seite 1 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "tzql@yywnjei-xxog.go",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Frauke",
    "Hegemann",
    "(Vorsitzende),",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Dr. Jochen",
    "Sutor"
  ],
  [
    "Kontoauszug Nr. 2020 / 1 und Rechnungsabschluss zum 31.03.2020",
    "EUR - Verrechnungskonto: 123456789",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "03.01.",
    "06.01.",
    "REF: 123456789",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 4,1985",
    "ISIN: LU0392494562",
    "ABR: 123456789",
    "03.01.",
    "06.01.",
    "REF: 123456789",
    "100,00-",
    "Wertpapierkauf",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "STK: 2,2499",
    "ISIN: LU0635178014",
    "ABR: 123456789",
    "06.01.",
    "06.01.",
    "REF: 123456789",
    "0,43-",
    "Überweisungausgang SEPA",
    "Umbuchung ishares",
    "YOEI_Eq: V821666130",
    "IBH_Ef: 4820999383",
    "27.01.",
    "27.01.",
    "REF: 123456789",
    "350,00+",
    "Überweisungseingang SEPA",
    "Umbuchung OnVista Sparplane - aktueller Monat",
    "BC90517030112005",
    "LKE946695172934748-928",
    "04.02.",
    "05.02.",
    "REF: 123456789",
    "100,00-",
    "Wertpapierkauf",
    "ABR: 123456789",
    "ISIN: LU0635178014",
    "STK: 2,3565",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "04.02.",
    "05.02.",
    "REF: 123456789",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 4,1738",
    "ISIN: LU0392494562",
    "ABR: 123456789",
    "25.02.",
    "25.02.",
    "REF: 123456789",
    "350,00+",
    "Überweisungseingang SEPA",
    "Umbuchung OnVista Sparplane - aktueller Monat",
    "AE34327769073806",
    "LKD190009563201074-714",
    "03.03.",
    "04.03.",
    "REF: 123456789",
    "100,00-",
    "Wertpapierkauf",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "STK: 2,446",
    "ISIN: LU0635178014",
    "ABR: 123456789",
    "Seite 2 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "ntcq@cqxbxih-cidt.ec",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Frauke",
    "Hegemann",
    "(Vorsitzende),",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Dr. Jochen",
    "Sutor",
    "Alter Kontostand vom 31.12.19",
    "350,43+"
  ],
  [
    "Kontoauszug Nr. 2020 / 1 und Rechnungsabschluss zum 31.03.2020",
    "EUR - Verrechnungskonto: 123456789",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "03.03.",
    "04.03.",
    "REF: 123456789",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 4,5075",
    "ISIN: LU0392494562",
    "ABR: 123456789",
    "16.03.",
    "16.03.",
    "REF: 123456789",
    "0,39+",
    "Zinsen/Dividenden",
    "ISHARES NASDAQ-100 U.ETF",
    "STK: 4,9438",
    "ISIN: DE000A0F5UF5",
    "ABR: 123456789",
    "25.03.",
    "25.03.",
    "REF: 123456789",
    "350,00+",
    "Überweisungseingang SEPA",
    "Umbuchung OnVista Sparplane - aktueller Monat",
    "YI71925291734863",
    "LPG691906196428802-974",
    "Seite 3 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "ijte@ttqwmft-prpb.mq",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Frauke",
    "Hegemann",
    "(Vorsitzende),",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Dr. Jochen",
    "Sutor",
    "Neuer Kontostand zum 31.03.2020",
    "350,39+"
  ]
]

```

</p></details>

## Community profile

@dredav

---
Issue submitted with https://import.tresor.one

**Comment by SirGibihm** [comment]
1) Will be implemented in #1062 
2) No valid activities contained within document, status code 5 will be thrown
3) Same as 2)
4) I implemented as gross === net in this case as we do not have any tax-information on these files anyways

**Comment by dredav** [comment]
- Oh I miss that there are no relevant transactions in 2. and 3.
- Your right with the dividend. Without any tax information there is no differents between gross and net.
- I found one more document (sorry I miss this one at the morning) with results in status code `3`: `Cannot read property 'split' of undefined`:

<details><summary>20181228_Kontoauszug.pdf</summary><p>

```json
[
  [
    "Kontoauszug Nr. 2018 / 4 zum 28.12.2018",
    "EUR - Verrechnungskonto: 1234567890",
    "Alter Saldo",
    "Neuer Saldo",
    "413,54",
    "H",
    "400,05",
    "H",
    "Guthaben",
    "sind als Einlagen",
    "nach",
    "Maßgabe",
    "des Einlagensicherungsgesetzes",
    "entschädigungsfähig.",
    "Nähere",
    "Informationen",
    "können",
    "dem",
    "\"Informationsbogen",
    "für den Einleger\",",
    "der unter",
    "www.onvista-bank.de",
    "im Menü",
    "\"Service",
    "- Formulare\"",
    "abgelegt",
    "ist, entnommen",
    "werden.",
    "Der",
    "Ihnen",
    "angezeigte",
    "Saldo",
    "kann",
    "Kontobewegungen",
    "berücksichtigen,",
    "deren",
    "tatsächliche",
    "Wertstellung",
    "(Valuta)",
    "noch",
    "nicht",
    "erfolgt",
    "ist.",
    "Bitte",
    "beachten",
    "Sie",
    "dies",
    "bei",
    "Ihrer",
    "Gelddisposition,",
    "da",
    "ansonsten",
    "Sollzinsen",
    "/ Überziehungszinsen",
    "anfallen",
    "könnten.",
    "Das",
    "Datum",
    "der",
    "Wertstellung",
    "(Valuta)",
    "können",
    "Sie",
    "auf",
    "den",
    "folgenden",
    "Seiten",
    "zur",
    "jeweiligen",
    "Buchung",
    "entnehmen.",
    "Einwendungen",
    "wegen",
    "Unrichtigkeit",
    "oder",
    "Unvollständigkeit",
    "eines",
    "Rechnungsabschlusses",
    "sind gem.",
    "Nr. 7",
    "Abs.",
    "2 unserer",
    "Allgemeinen",
    "Geschäftsbedingungen",
    "spätestens",
    "vor Ablauf",
    "von sechs",
    "Wochen",
    "nach",
    "Zugang",
    "des Rechnungsabschlusses",
    "zu erheben.",
    "Sofern",
    "Sie Ihre Einwendungen",
    "schriftlich",
    "geltend",
    "machen,",
    "richten",
    "Sie",
    "diese",
    "bitte",
    "unter",
    "Angabe",
    "der",
    "oben",
    "genannten",
    "Kontonummer",
    "an",
    "die",
    "onvista",
    "bank,",
    "Interne",
    "Revision,",
    "Wildunger",
    "Straße",
    "6a,",
    "60487",
    "Frankfurt",
    "am",
    "Main.",
    "Es genügt",
    "die Absendung",
    "innerhalb",
    "der Sechs-Wochen-Frist.",
    "Das",
    "Unterlassen",
    "rechtzeitiger",
    "Einwendungen",
    "gilt",
    "als",
    "Genehmigung.",
    "Bei Zahlungen",
    "größer",
    "12.500",
    "EUR",
    "bitte Meldepflichten",
    "nach",
    "dem Außenwirtschaftsrecht",
    "beachten.",
    "Unsere Postanschrift:",
    "onvista bank",
    "Postfach 10 08 60",
    "60008 Frankfurt",
    "Kundenservice:",
    "Telefon:",
    "+49 (0) 69 7107 - 530",
    "Fax",
    "+49 (0) 69 7107 - 100",
    "Mo-Fr 8:00 - 19:00 Uhr",
    "jpbqtai@hhtcvff-xwav.rr",
    "Ihre Bankverbindung für Überweisungen:",
    "IBAN",
    "XF34327233238988677682",
    "BIC",
    "BOURDEFF",
    "Herr",
    "Fplvz Sofhzy",
    "Cr Apekgaerzyrgk 43",
    "15604 Wfccrbua Xz Pphcuooq",
    "Seite 1 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "vyow@wullkfx-faug.nl",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Arno",
    "Walter",
    "(Vorsitzender),",
    "Dietmar",
    "von",
    "Blücher,",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Michael",
    "Mandel"
  ],
  [
    "Kontoauszug Nr. 2018 / 4 zum 28.12.2018",
    "EUR - Verrechnungskonto: 1234567890",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "02.10.",
    "03.10.",
    "REF: 1234567890",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 4,7388",
    "ISIN: LU0392494562",
    "ABR: 1234567890",
    "02.10.",
    "03.10.",
    "REF: 1234567890",
    "100,00-",
    "Wertpapierkauf",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "STK: 2,5201",
    "ISIN: LU0635178014",
    "ABR: 1234567890",
    "03.10.",
    "03.10.",
    "REF: 1234567890",
    "63,54-",
    "Überweisungausgang SEPA",
    "Umbuchung",
    "KQVG_Qp: Y881630376",
    "HJQ_Hz: 115992750",
    "22.10.",
    "22.10.",
    "REF: 1234567890",
    "350,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista - Festpreis Depot",
    "NOTPROVIDED",
    "LAY637973779968056-737",
    "02.11.",
    "05.11.",
    "REF: 1234567890",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 5,031",
    "ISIN: LU0392494562",
    "ABR: 1234567890",
    "02.11.",
    "05.11.",
    "REF: 1234567890",
    "100,00-",
    "Wertpapierkauf",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "STK: 2,7022",
    "ISIN: LU0635178014",
    "ABR: 1234567890",
    "22.11.",
    "22.11.",
    "REF: 1234567890",
    "400,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista - Festpreis Depot",
    "NOTPROVIDED",
    "LJM193348799021838-190",
    "04.12.",
    "05.12.",
    "REF: 1234567890",
    "50,00-",
    "Wertpapierkauf",
    "ISHARES NASDAQ-100 U.ETF",
    "STK: 0,8176",
    "ISIN: DE000A0F5UF5",
    "ABR: 1234567890",
    "Seite 2 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "esxu@ckozmpm-mxva.hv",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Arno",
    "Walter",
    "(Vorsitzender),",
    "Dietmar",
    "von",
    "Blücher,",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Michael",
    "Mandel",
    "Alter Kontostand vom 28.09.18",
    "413,54+"
  ],
  [
    "Kontoauszug Nr. 2018 / 4 zum 28.12.2018",
    "EUR - Verrechnungskonto: 1234567890",
    "Buchungs-",
    "Valuta",
    "Buchungstext",
    "Soll in EUR",
    "Haben in EUR",
    "datum",
    "04.12.",
    "05.12.",
    "REF: 1234567890",
    "250,00-",
    "Wertpapierkauf",
    "COMS.-MSCI WORL.T.U.ETF I",
    "STK: 4,8708",
    "ISIN: LU0392494562",
    "ABR: 1234567890",
    "04.12.",
    "05.12.",
    "REF: 1234567890",
    "100,00-",
    "Wertpapierkauf",
    "COMS.-MSCI EM.M.T.U.ETF I",
    "STK: 2,5347",
    "ISIN: LU0635178014",
    "ABR: 1234567890",
    "17.12.",
    "17.12.",
    "REF: 1234567890",
    "0,05+",
    "Zinsen/Dividenden",
    "ISHARES NASDAQ-100 U.ETF",
    "STK: 0,8176",
    "ISIN: DE000A0F5UF5",
    "ABR: 1234567890",
    "27.12.",
    "27.12.",
    "REF: 1234567890",
    "400,00+",
    "Überweisungseingang SEPA",
    "Umbuchung onvista - Festpreis Depot",
    "NOTPROVIDED",
    "LET172590091076180-057",
    "Seite 3 von 3",
    "onvista",
    "bank",
    "Wildunger",
    "Straße",
    "6a",
    "60487",
    "Frankfurt",
    "am",
    "Main",
    "T: +49",
    "(0)69-7107-0",
    "F: +49",
    "(0)69-7107-100",
    "ecmb@nnlyrpj-jswa.tq",
    "www.onvista-bank.de",
    "onvista",
    "bank",
    "ist eine",
    "Marke",
    "der",
    "comdirect",
    "bank",
    "AG",
    "Sitz",
    "der",
    "comdirect",
    "bank",
    "AG:",
    "Pascalkehre",
    "15",
    "25451",
    "Quickborn",
    "(AG",
    "Pinneberg",
    "HRB",
    "4889)",
    "T: +49",
    "(0)4106-704-0",
    "F: +49",
    "(0)4106-708-258-5",
    "www.comdirect.de",
    "Vorstand:",
    "Arno",
    "Walter",
    "(Vorsitzender),",
    "Dietmar",
    "von",
    "Blücher,",
    "Matthias",
    "Hach",
    "Vorsitzender",
    "des",
    "Aufsichtsrates:",
    "Michael",
    "Mandel",
    "Neuer Kontostand zum 28.12.2018",
    "400,05+"
  ]
]
```

</p></details>
